### PR TITLE
prepare for 2.5.6 release

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ defaults:
     # note that changing this requires a jekyll restart to be picked up if serving locally
     # versions.json will be updated as long as the changes here are patch versions, for minor and major it needs to
     # be updated manually
-    current_akka_version: 2.5.4
+    current_akka_version: 2.5.6
     current_akka_http_version: 10.0.10
     previous_akka_version: 2.4.20
-    current_java_scala_version: 2.11 # used for deps for Java
+    current_java_scala_version: 2.12 # used for deps for Java
```
Comment (ktoso) [inline _config.yml]: since provides better lambda interop?

File: blog/_posts/2017-09-28-akka-2.5.6-released.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Akka HTTP 2.5.5 Released – with JDK9 and j.u.c.Flow support
+title: Akka HTTP 2.5.6 Released – with JDK9 and j.u.c.Flow support
 author: Konrad 'ktoso' Malawski
```
Comment (ktoso) [inline blog/_posts/2017-09-28-akka-2.5.6-released.md]: and renamed the file

File: blog/_posts/2017-09-28-akka-2.5.6-released.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Akka HTTP 2.5.6 Released – with JDK9 and j.u.c.Flow support
+author: Konrad 'ktoso' Malawski
+short: We are happy to announce a new patch release of Akka 2.5
+category: news
+tags: [releases]
```
Comment (ktoso) [inline blog/_posts/2017-09-28-akka-2.5.6-released.md]: new news item for 2.5.6

File: _config.yml (hunk 1 above)
Comment (ktoso) [reply _config.yml]: cc @patriknw @akka/akka-team

Comment (raboof) [reply _config.yml]: Makes sense to recommend that I suppose, yes. Not sure why we didn't before?

File: blog/_posts/2017-09-28-akka-2.5.6-released.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.5. 
+Other than the usual hardening and maintenance work, during this period we made a lot of progress on multiple very important new features which we’re very proud to release today.
```
Comment (raboof) [inline blog/_posts/2017-09-28-akka-2.5.6-released.md]: Maybe spelling out 'we are' looks more professional?

Comment (ktoso) [reply blog/_posts/2017-09-28-akka-2.5.6-released.md]: ok

File: _config.yml (hunk 1 above)
Comment (ktoso) [reply _config.yml]: no idea :)

Comment (patriknw) [reply _config.yml]: ok, might have been because of the fjp regression, but that's fixed

File: blog/_posts/2017-09-28-akka-2.5.6-released.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Akka HTTP 2.5.6 Released – with JDK9 and j.u.c.Flow support
```
Comment (patriknw) [inline blog/_posts/2017-09-28-akka-2.5.6-released.md]: remove `HTTP`

**Pull request merged**